smaug/python/graph.py

```diff
@@ -71,6 +73,12 @@ def add_node(
     if params != None:
       node.params.CopyFrom(params)
 
+    # If we are currently within a control flow context, add the new node to
+    # the context.
+    if isinstance(self._control_flow_context,
```

Comment:
Why is comparing this to None not sufficient?

Reply:
Changed to comparing to None.